docs/index.rst

```diff
@@ -23,43 +23,43 @@ The Tools
 The tools are accessible from both a command-line interface (CLI) and a Python API. The CLI automates many operations commonly needed in NWP workflows. The API supports all CLI operations, and additionally provides access to in-memory objects to facilitate more novel use cases. We hope these options will let you integrate the package into your pre-existing bash and Python scripts, and give you some handy tools to use in your day-to-day work with running NWP systems.
 
 Configuration Management
-~~~~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
There's a note on the _Documentation_ page of the _Developer Guide_ now explaining this, but: The Sphinx docs provide a recommendation for [[sub]sub]section markup and, to keep our formatting consistent for all contributors, I've recommended that we use that standard. I've tried to update everything to match as well.